flattentool/cli.py

```diff
@@ -222,19 +228,30 @@ def main():
 
     if args.subparser_name is None:
         parser.print_help()
-    elif args.subparser_name == 'create-template':
-        # Pass the arguments to the create_template function
-        # If the schema file does not exist we catch it in this exception
-        try:
-            # Note: Ensures that empty arguments are not passed to the create_template function
-            create_template(**kwargs_from_parsed_args(args))
-        except (OSError, IOError) as e:
-            print(text_type(e))
-            return
-    elif args.subparser_name == 'flatten':
-        flatten(**kwargs_from_parsed_args(args))
-    elif args.subparser_name == 'unflatten':
-        unflatten(**kwargs_from_parsed_args(args))
+    else:
+        if not args.verbose:
+            def custom_warning_formatter(message, category, filename, lineno, line=None):
```

Comment:
Also I have a style preference for not having nested functions (even if single use) if not needed for closures.  I do not not see why this function can not live elsewhere either top or bottom of file (calling it maybe terse_warning_formatter)

Reply:
The closure is needed here to refer to `default_warning_formatter`. If moved, we'd need a method that takes the default formatters as an argument and returns a method, which seems to increase complexity without clear benefit.